src/index.js

```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import Routes from "./Routes";
+
+import ToggleButton from "./components/Darkmode/ToggleButton";
```

Comment:
I think you should name `ToggleButton` to more specific name. like `DarkModeToggleButton`.

Reply:
Oh, that's a more intuitive name! Naming is difficult... 
This pr has already been merged, so I'll merge all the parts you mentioned and post it in the next pr. 
I wish you comment on the next pr 👻🌠